analysis/lang/pl/stemmer_pl.go

```diff
@@ -0,0 +1,56 @@
+//  Copyright (c) 2018 Couchbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pl
+
+import (
+	"github.com/blevesearch/bleve/v2/analysis"
+	"github.com/blevesearch/bleve/v2/analysis/lang/pl/stempel"
+	"github.com/blevesearch/bleve/v2/registry"
+	"log"
+)
+
+const SnowballStemmerName = "stemmer_pl"
+
+type PolishStemmerFilter struct {
+	trie stempel.Trie
+}
+
+func NewPolishStemmerFilter() *PolishStemmerFilter {
+	trie, err := stempel.LoadTrie()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Let's return an error here as opposed to log.Fatal.

Reply:
done, thank you for review